test/xain_test.exs

```diff
@@ -298,4 +298,16 @@ defmodule XainTest do
     end
     assert result == ~s(<div></div>)
   end
+
```

Comment:
I'd like to see a couple more tests here. Specifically check that we don't break syntax like: `div "#id.my-class <hello>"`